experimental/dds/tree2/src/core/rebase/utils.ts

```diff
@@ -79,6 +84,8 @@ export interface RebasedCommits<TChange> {
  */
 export function rebaseBranch<TChange>(
 	changeRebaser: ChangeRebaser<TChange>,
+	intoDelta: (change: TChange) => Delta.Root,
```

Comment:
We need this so that we can update the `IRepairDataStoreProvider`. Now that providers are somewhat coupled with changes (because they go in commits), what if we had `IRepairDataStoreProviders` take a `changeFamily` and their API would deal in _changes_ rather than deltas? The provider would simply generate the delta itself internally (at least, the ForestProvider would; if we had an implementation of `IRepairDataStoreProvider` that wasn't backed by a forest, would we necessarily event need/want deltas?). That would clean up this parameter and all the call sites, though I'm not sure if it would muddy anything else. Worth a try though?

Reply:
Seems worth a shot. I fear it may lead to the same delta being computed multiple times, but I'd rather pay that cost for the sake clarity.

Reply:
So it turns out this parameter is still needed because `RepairDataStore` takes a delta for the capture call. I think it should be fine to also store change family in there but can you confirm that's ok, @yann-achard-MS ?

Reply:
Oh yea, sorry I wasn't clear. I was imagining that repair data stores would probably have the `changeFamily` provided at construction time and would keep it as a member variable (as many of our classes all over the place do).

Reply:
Yep, storing a change family in the repair data store should be OK.

Reply:
Although it may be preferable to just pass the `intoDelta: (change: TChange) => Delta.Root` function itself, since that's all the repair data store needs. This will make testing easier since you'll be able to have `TChange` just be `Delta.Root`.